extensions/distributed-tracing.md

```diff
@@ -24,25 +24,31 @@ Prometheus are built.
 - Constraints
   - OPTIONAL
 
-## HTTP encoding
+## Using the Distributed Tracing Extension
 
-To integrate with existing tracing libraries, the Distributed Tracing attributes
-MUST be encoded over HTTP(S) as headers. E.g.
+The Distributed Tracing Extension is not intended to replace the protocol specific headers for tracing, 
+like the ones described in [W3C Trace Context](https://w3c.github.io/trace-context/) for HTTP.
+
+Given a single hop event transmission (from sink to source directly), the Distributed Tracing Extension, 
+if used, MUST carry the same trace information contained in protocol specific tracing headers.
+
+Given a multi hop event transmission, the Distributed Tracing Extension, if used, MUST 
+carry the trace information of the starting trace of the transmission. 
+In other words, it MUST NOT carry trace information of the actual hop, since these information are usually 
```

Comment:
"of the actual hop" might be confusing.
Did you mean: "of each individual hop"   ?

Also: s/these information are/is information is/